app/views/facilities/edit.html.erb

```diff
@@ -1,25 +1,34 @@
 <%=form_for @facility do |f|%>
-    
-  
-    <div>
-        <%=f.text_field  :name ,placeholder:"Name of Facility"%> 
-    </div>
-    
-    <div>
-        <%=f.text_field :city ,placeholder:"City"%> 
-    </div>
-    <div>
-        <%=f.text_field :state ,placeholder:"State"%> 
-    </div>
-    <div>
-        <%=f.text_field :zip ,placeholder:"ZIP Code"%> 
-    </div>
-    <div>
-        <%=f.text_field :specialty ,placeholder:"Speciality of Facility"%> 
-    </div>
-    
-     <div>
-        <%=f.submit "Post"%>
-    </div>
 
-<%end%>
\ No newline at end of file
+  <div>
+    <%= f.text_field  :name, placeholder: "Name of Facility" %>
+    <%= f.label :facility, 'Facility' %>
```

Comment:
If the string is the same as the actual parameter name, then you don't need to specify it.